cmds/coredhcp/main.go

```diff
@@ -15,14 +15,15 @@ import (
 	"github.com/coredhcp/coredhcp/server"
 
 	"github.com/coredhcp/coredhcp/plugins"
-	"github.com/coredhcp/coredhcp/plugins/dns"
-	"github.com/coredhcp/coredhcp/plugins/file"
-	"github.com/coredhcp/coredhcp/plugins/leasetime"
-	"github.com/coredhcp/coredhcp/plugins/nbp"
-	"github.com/coredhcp/coredhcp/plugins/netmask"
-	rangepl "github.com/coredhcp/coredhcp/plugins/range"
-	"github.com/coredhcp/coredhcp/plugins/router"
-	"github.com/coredhcp/coredhcp/plugins/serverid"
+	pl_dns "github.com/coredhcp/coredhcp/plugins/dns"
```

Comment:
I'd like to have a comment here to explain why the `pl_` prefix is necessary, it would be confusing otherwise

Reply:
How about a "this is a generated file" header, like:
```
// This is a generated file, based on cmds/coredhcp-generator/coredhcp.go.template
// Regenerate with `coredhcp-generator -from core-modules.txt`
```
This doesn't directly address the plugin names but any weirdness will be explained by looking at the source template ?

Reply:
yes, sounds good